src/com/amazon/kinesis/streaming/agent/Agent.java

```diff
@@ -52,8 +52,10 @@ public static void main(String[] args) throws Exception {
         AgentOptions opts = AgentOptions.parse(args);
         String configFile = opts.getConfigFile();
         AgentConfiguration config = tryReadConfigurationFile(Paths.get(opts.getConfigFile()));
-        Path logFile = opts.getLogFile() != null ? Paths.get(opts.getLogFile()) : (config != null ? config.logFile() : null);
-        String logLevel = opts.getLogLevel() != null ? opts.getLogLevel() : (config != null ? config.logLevel() : null);
+
+        //Path logFile = opts.getLogFile() != null ? Paths.get(opts.getLogFile()) : (config != null ? config.logFile() : null);
+        Path logFile = config != null ? config.logFile() : (opts.getLogFile() != null ? Paths.get(opts.getLogFile()) : null);
```

Comment:
This specifies that configs should be loaded from the config file first rather than from the options, it's consistent with the same fix that we made in line 58